tests/gla/block_mat_02.with_trilinos=true.mpirun=1.output.2

```diff
@@ -0,0 +1,769 @@
+
+DEAL:0:PETSc::numproc=1
+{[0,40]}
+Block (0, 0)
+(0,0) (0.00000,0.00000)
+(0,1) (0.00000,0.00000)
```

Comment:
Is it expected, by the way, that all of these blocks of the matrix are zero? It seems to me like everything here is just zero.